### DISCUSSION TITLE
License Change to Apache 2.0

### DISCUSSION THREAD

**Discussion by harold-b** [Chia-Network/bladebit#95] [Announcements]
I am planning on changing the license to a more permissive [Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0) to be in-line with Chia as we're going to more closely integrate it with the official tools.

@mocksoul since you've contributed PRs I wanted to inform you and make sure you're OK with the license change.

**Comment by harold-b** [comment]
See PR https://github.com/harold-b/bladebit/pull/96

**Reply by mocksoul** [reply]
sure, thnx for asking
you made this, not me )

**Comment by harold-b** [comment]
Great :)
It has been merged.
